_components/header/header.html

```diff
@@ -23,7 +23,7 @@
     href: '#extended-header-with-megamenu'
 ---
 
-<h3>When to use the header component</h3>
+<h2>When to use the header component</h2>
```

Comment:
Fixes throughout that address heading level. In general, body text begins at the `h2` level